modules/enumeratum/README.md

```diff
@@ -42,7 +47,6 @@ val conf = parseString("""{
   end: SHOUT_GOOD_BYE
 }""")
 loadConfig[GreetingConf](conf)
```

Comment:
I don't understand why the result of `loadConfig` didn't get printed here. It seems like it should've. Any ideas?